x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/epm/screens/home/index.tsx

```diff
@@ -141,7 +145,13 @@ function AvailablePackages() {
       isLoading={isLoadingCategories}
       categories={categories}
       selectedCategory={selectedCategory}
-      onCategoryChange={({ id }: CategorySummaryItem) => setSelectedCategory(id)}
+      onCategoryChange={({ id }: CategorySummaryItem) => {
+        // clear category query param in the url
+        if (queryParams.get('category') !== null) {
+          history.push({});
+        }
```

Comment:
@parkiino is `null` significant here? Can we rely on "truthy"?
```suggestion
        if (queryParams.get('category')) {
```

cc @jen-huang and @neptunian for thoughts on dealing with the params/state